kola/harness.go

```diff
@@ -377,12 +377,17 @@ func getClusterSemver(pltfrm, distro, outputDir string) (*semver.Version, error)
 		return nil, fmt.Errorf("parsing /etc/os-release: %v: %s", err, stderr)
 	}
 
-	version, err := semver.NewVersion(strings.Split(string(out), "=")[1])
-	if err != nil {
-		return nil, fmt.Errorf("parsing os-release semver: %v", err)
+	// TODO: add distro specific version handling
+	if distro != "rhcos" {
```

Comment:
I would prefer an
```
if distro == "rhcos" {
    return &semver.Version{}
}
```
to keep indentation minimal

Reply:
Would you prefer if I made this:
```golang
if distro == "cl" {
    ...
}
```

I think in the end we'll end up with this kind of indentation once we start handling other version parsing.

Reply:
Actually, what about breaking this into `getCLVersion` and `getRHCOSVersion` and use a switch. Would make adding FCOS later easier as well.